docs/source/markdown/options/ignore.md

```diff
@@ -1,9 +1,9 @@
 ####> This option file is used in:
-####>   podman pod rm, pod stop, rm, stop
+####>   podman pod rm, pod stop, rm, secret rm, stop
 ####> If file is edited, make sure the changes
 ####> are applicable to all of those.
 #### **--ignore**, **-i**
 
-Ignore errors when specified <<containers|pods>> are not in the container store.  A user
-might have decided to manually remove a <<container|pod>> which leads to a failure
-during the ExecStop directive of a systemd service referencing that <<container|pod>>.
+Ignore errors when specified <<containers|pods|secrets>> are not in the container store.  A user
+might have decided to manually remove a <<container|pod|secret>> which leads to a failure
+during the ExecStop directive of a systemd service referencing that <<container|pod|secret>>.
```

Comment:
Sorry, it doesn't work this way. This only works for containers or pods.

Reply:
...and after looking at `hack/markdown-preprocess`, it is not reasonably possible to make this work. I'm afraid you'll have to revert the `podman-secret-rm` rename and manually document `--ignore`. Which, honestly, is the right thing to do because "...not in the container store" does not make sense in the context of secrets.

Reply:
Thanks for the review, should've noticed the discrepancy in the text.